.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+<!--- ↑ Provide a general summary of your changes in the Title above ↑ -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Related Issue
+
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an -->
+<!--- issue first -->
+<!--- If fixing a bug, there should be an issue describing it with -->
+<!--- steps to reproduce -->
+<!--- Please link to the issue here with the prefix or resolve -->
+<!--- keyword -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests -->
+<!--- you ran to see how your change affects other areas of the -->
+<!--- code, etc. -->
+
+## Screenshots
+
+<!--- if appropriate, otherwise the section can be removed -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Stroke -->
+<!-- (i.e. ~stroked text~) the ones that don't apply: -->
+
+- Chore (non-breaking change which refactors / improves the existing code base).
+- Bug fix (non-breaking change which fixes an issue).
+- New feature (non-breaking change which adds functionality).
+- Breaking change (fix or feature that would cause existing functionality to
+  change).
+
+## Checklist:
+
+<!--- Go over all the following points, and replace the `:red_circle:` in all -->
+<!--- the lines with a :white_check_mark: when relevant. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're -->
+<!--- here to help! -->
+
+- :red_circle: My code follows the code style of this project.
+- :red_circle: My change requires a change to the documentation.
+- :red_circle: I have updated the documentation accordingly.
+- :red_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
+- :red_circle: I have added tests to cover my changes.
+- :red_circle: All new and existing tests passed.
+
+[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md
```

Comment:
Let's make it the contributing of this repo since `connect-account` is public

Reply:
You're right. I'm also wondering it there is any use for the `CONTIRBUTING.md` file.

Reply:
not at the moment lest we split your README in two ^^